src/Razor/test/Microsoft.AspNetCore.Razor.LanguageServer.Test/Semantic/DefaultRazorSemanticTokenInfoServiceTest.cs

```diff
@@ -61,7 +61,25 @@ public void GetSemanticTokens_IgnoresNonTagHelperAttributes()
                 0, 39, 5, 0, 0
             };
 
-            AssertSemanticTokens(txt, expectedData);
+            AssertSemanticTokens(txt, expectedData, isRazor: false);
+        }
+
+        [Fact]
+        public void GetSemanticTokens_DoesNotFindCapitalizedTagHelpers()
+        {
+            var txt = $"@addTagHelper *, TestAssembly{Environment.NewLine}<TestElement bool-val='true' class='display:none'></TestElement>";
+            var expectedData = new List<uint> { };
+
+            AssertSemanticTokens(txt, expectedData, isRazor: false);
+        }
+
+        [Fact]
+        public void GetSemanticTokens_CasingMattersInRazor()
```

Comment:
Can you be more specific in this test name. Not entirely sure which piece it's testing